SpectatorView/Dependencies/OpenCV/include/opencv2/calib3d/calib3d.hpp

```diff
@@ -0,0 +1,48 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Do not redistribute the OpenCV source - install directory is in dependencies.props so we do not redistribute